commands/previewPush.go

```diff
@@ -216,6 +228,65 @@ DomainLoop:
 		totalCorrections += len(corrections)
 		anyErrors = printOrRunCorrections(domain.Name, domain.RegistrarName, corrections, out, push, interactive, notifier) || anyErrors
 	}
+
+	managedDomains := make([]string, 0, len(cfg.Domains))
+	for _, zone := range cfg.Domains {
+		managedDomains = append(managedDomains, zone.Name)
+	}
+
+	if args.DeleteUnmanagedDomains {
+		fmt.Printf("Checking Domain Removal:\n")
+		if len(createdRegistrars) != 1 {
+			return fmt.Errorf("To delete unmanaged domains, exactly one registrar must be used")
```

Comment:
why is that a requirement?

Reply:
My mistake, this should not be required.